README.md

```diff
@@ -71,7 +71,8 @@ Please refer to the following instructions to setup Oppiabot for the first time
 
 ## Installing the bot on a repository
 
-You'll need to identify a target repository and install the bot by clicking the **Install** button on the settings page of your app, e.g `https://github.com/apps/your-app-name`
+You'll need to identify a target repository and install the bot by clicking the **Install** button on the settings page of your app, e.g `https://github.com/apps/your-app-name`. In the `.env` file put your github account name in 
+`WHITELISTED_ACCOUNTS` and also add your repository (name in small caps) in the `constants.js` file locally.
```

Comment:
> also add your repository (name in small caps) in the `constants.js` file locally

Why is this required?

Reply:
I think we need to add the name of our repository along with the events in constants.js file for running oppiabot in the repository, as I did that too.